File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans = []

        for i in range(1, n+1):
            a = i%3
            b = i%5

            if a + b == 0:
                ans.append("FizzBuzz")
            elif a == 0:
                ans.append("Fizz")
            elif b == 0:
                ans.append("Buzz")
            else:
                ans.append(str(i))
        
        return ans

File: 0412-fizz-buzz/0412-fizz-buzz.swift
class Solution {
    func fizzBuzz(_ n: Int) -> [String] {
        var ans: [String] = []

        for i in 1...n {
            var tempStr: String = ""

            if i%3 == 0 {
                tempStr += "Fizz"
            }

            if i%5 == 0 {
                tempStr += "Buzz"
            }

            if tempStr == "" {
                tempStr = String(i)
            }
            
            ans.append(tempStr)
        }
        
        return ans
    }
}